src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudWindowsIT.java

```diff
@@ -0,0 +1,299 @@
+    private static final String NAME_PREFIX = "integration";
+    private static final String REGION = format("projects/%s/regions/us-west1");
+    private static final String ZONE = "us-west1-a";
+    private static final String ZONE_BASE = format("projects/%s/zones/" + ZONE);
+    private static final String LABEL = "integration";
+    private static final String MACHINE_TYPE = ZONE_BASE + "/machineTypes/n1-standard-1";
+    private static final String NUM_EXECUTORS = "1";
+    private static final boolean PREEMPTIBLE = false;
+    private static final String CONFIG_DESC = "integration";
+    private static final String MIN_CPU_PLATFORM = "Intel Broadwell";
+    private static final String BOOT_DISK_TYPE = ZONE_BASE + "/diskTypes/pd-ssd"; //hmm
+    private static final boolean BOOT_DISK_AUTODELETE = true;
+    private static final String BOOT_DISK_SIZE_GB_STR = "50";
+    private static final Node.Mode NODE_MODE = Node.Mode.EXCLUSIVE;
+    private static final String ACCELERATOR_NAME = "";
+    private static final String ACCELERATOR_COUNT = "";
+    private static final String RUN_AS_USER = "jenkins";
+
+    private static final String WINDOWS_USER = "Build";
+
+    private static Map<String, String> INTEGRATION_LABEL;
+
+    static {
+        INTEGRATION_LABEL = new HashMap<String, String>();
+        INTEGRATION_LABEL.put("integration", "delete");
+    }
+
+    private static final String NETWORK_NAME = format("projects/%s/global/networks/default");
+    private static final String SUBNETWORK_NAME = "default";
+    private static final boolean EXTERNAL_ADDR = true;
+    private static final String NETWORK_TAGS = "ssh";
+    private static final String SERVICE_ACCOUNT_EMAIL = "";
+    private static final String RETENTION_TIME_MINUTES_STR = "600";
+    private static final String LAUNCH_TIMEOUT_SECONDS_STR = "3000";
+
+    private static Logger cloudLogger;
+    private static Logger clientLogger;
+    private static StreamHandler sh;
+    private static ByteArrayOutputStream logOutput;
+
+    private static ComputeClient client;
+    private static String projectId;
+    private static String bootDiskProjectId;
+    private static String bootDiskImageName;
+
+    private static String format(String s) {
+        String projectId = System.getenv("GOOGLE_PROJECT_ID");
+        if (projectId == null) {
+            throw new RuntimeException("GOOGLE_PROJECT_ID env var must be set");
+        }
+        return String.format(s, projectId);
+    }
+
+    @ClassRule
+    public static JenkinsRule r = new JenkinsRule();
+
+    @BeforeClass
+    public static void init() throws Exception {
+        log.info("init");
+        logOutput = new ByteArrayOutputStream();
+        sh = new StreamHandler(logOutput, new SimpleFormatter());
+
+        // Add a service account credential
+        projectId = System.getenv("GOOGLE_PROJECT_ID");
+        assertNotNull("GOOGLE_PROJECT_ID env var must be set", projectId);
```

Comment:
Assertions should be reserved for test cases to preserve test readability. Suggest using Precondition.checkNotNull() instead.

Reply:
Here and throughout the init() method.

Reply:
Will take care of this in refactor since the other IT does the same.